metric/export.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metric
+
+import (
+	"go.opencensus.io/metric/metricdata"
+)
+
+// Exporter is an interface that exporters implement to export the metric data.
+type Exporter interface {
+	ExportMetric(data []*metricdata.Metric)
```

Comment:
this conflicts with already defined [interface](https://github.com/census-ecosystem/opencensus-go-exporter-stackdriver/blob/ab5a58af316a529613aadf9f50eeed1b6f044b2f/stackdriver.go#L349) used for OC-service.
Should this be changed to ?
1. Export( ) OR
2. ExportMetrics() . - It is plural, could be very confusing.

OR should the oc-service change it to
1. ExportMetricPb(...) -  because it exports metric protobuf 

@bogdandrutu, @songy23 any thoughts?

Reply:
> should the oc-service change it to

> ExportMetricPb(...) - because it exports metric protobuf

I would suggest to go with this approach. I think we should stick to "**Metric**" when we're referring to the native Go data models and "**MetricPb**" when referring to the proto-generated files.

Reply:
/cc @odeke-em